product_pricelist_fixed_currency_rate/tests/test_product_pricelist_fixed_currency_rate.py

```diff
@@ -12,85 +12,97 @@ def setUpClass(cls):
         cls.product_model = cls.env["product.product"]
 
         cls.company = cls.env["res.company"].create(
-            {
-                "name": "Test",
-                "currency_id": cls.env.ref("base.EUR").id,
-            }
+            [
+                {
+                    "name": "Test",
+                    "currency_id": cls.env.ref("base.EUR").id,
+                }
+            ]
```

Comment:
IMO this change (in the whole test file) is not required (`create` accepts a dictionary), and introduces diff that could be annoying when porting commits.